server/routes/makeAReferral/makeAReferralController.ts

```diff
@@ -496,16 +497,37 @@ export default class MakeAReferralController {
 
   async viewRiskInformation(req: Request, res: Response): Promise<void> {
     const referral = await this.interventionsService.getDraftReferral(res.locals.user.token.accessToken, req.params.id)
-    const [serviceUser, riskSummary] = await Promise.all([
-      this.communityApiService.getServiceUserByCRN(referral.serviceUser.crn),
-      this.assessRisksAndNeedsService.getRiskSummary(referral.serviceUser.crn, res.locals.user.token.accessToken),
-    ])
+    const serviceUser = await this.communityApiService.getServiceUserByCRN(referral.serviceUser.crn)
+    if (config.apis.assessRisksAndNeedsApi.riskSummaryEnabled) {
+      await this.displayOasysRiskInformationPage(res, referral, serviceUser)
+    } else {
+      await this.displayRiskInformationPage(res, referral, serviceUser)
```

Comment:
Again, could we possibly distinguish these two pages a bit more clearly in their names? `OasysRiskInformationPage` vs `RiskInformationPage` might not be that obvious to readers of the code. I suppose this old route will eventually be deprecated, but could we call it something like `displayInterventionsRiskInformationPage` (or something less bad than that, I'm struggling to think of a better representation, sorry 😄 )